eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -75,10 +75,7 @@ jobs:
       vmImage: $(OSVmImage)
 
     steps:
-      # https://github.com/actions/virtual-environments/issues/798
-      - script: sudo ln -sf /run/systemd/resolve/resolv.conf /etc/resolv.conf
-        displayName: Bypass local DNS server to workaround issue resolving cognitiveservices names
-        condition: and(succeededOrFailed(), eq(variables['OSVmImage'], 'ubuntu-18.04'))
+      - template: ../../../common/pipelines/templates/steps/bypass-local-dns.yml
```

Comment:
`/eng/common/pipelines/templates/steps/bypass-local-dns.yml`

Reply:
Thanks. I will make this change.